var/spack/repos/builtin/packages/ispc/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+# ispc requires <gnu/stubs-32.h>, e.g. from
+# glibc-devel.i686 (CentoOS) or libc6-dev-i386 and g++-multilib (Ubuntu)
+
+
+from spack import *
+
+
+class Ispc(CMakePackage):
+    """Intel Implicit SPMD Program Compiler
+
+    An open-source compiler for high-performance SIMD programming on the CPU"""
+
+    homepage = "https://ispc.github.io"
+    url      = "https://github.com/ispc/ispc/tarball/v1.14.1"
+    maintainers = ['aumuell']
+
+    version('1.14.1', sha256='ca12f26dafbc4ef9605487d03a2156331c1351a4ffefc9bab4d896a466880794')
+    version('1.14.0', sha256='1ed72542f56738c632bb02fb0dd56ad8aec3e2487839ebbc0def8334f305a4c7')
+    version('1.13.0', sha256='aca595508b51dd1ff065c406a3fd7c93822320c510077dd4d97a2b98a23f097a')
+
+    depends_on('python', type='build')
+    depends_on('bison', type='build')
+    depends_on('flex', type='build')
+    depends_on('ncurses+termlib', type='link')
+    depends_on('zlib', type='link')
+    depends_on('llvm~libcxx')
```

Comment:
Should this also be `type='build'`?

Reply:
I don't think that `type='build'` is sufficient, as the target binary `ispc` is linked against LLVM libraries: `-I` and `-L` flags will be required, although it links statically against LLVM, so there is no run-time dependency.

Reply:
In that case you should remove `type='build'` from the next two lines.

Reply:
makes sense that these should be consistent, PR updated accordingly